src/debugging.js

```diff
@@ -0,0 +1,89 @@
+
+import { config } from 'src/config';
+import { logMessage as utilsLogMessage, logWarn as utilsLogWarn } from 'src/utils';
+import { addBidResponse } from 'src/auction';
+
+const OVERRIDE_KEY = '$$PREBID_GLOBAL$$:debugging';
+
+export let boundHook;
+
+function logMessage(msg) {
+  utilsLogMessage('DEBUG: ' + msg);
+}
+
+function logWarn(msg) {
+  utilsLogWarn('DEBUG: ' + msg);
+}
+
+function enableOverrides(overrides, fromSession = false) {
+  config.setConfig({'debug': true});
+  logMessage(`bidder overrides enabled${fromSession ? ' from session' : ''}`);
+
+  if (boundHook) {
+    addBidResponse.removeHook(boundHook);
+  }
+
+  boundHook = addBidResponseHook.bind(null, overrides);
+  addBidResponse.addHook(boundHook, 5);
+}
+
+export function disableOverrides() {
+  if (boundHook) {
+    addBidResponse.removeHook(boundHook);
+    logMessage('bidder overrides disabled');
+  }
+}
+
+export function addBidResponseHook(overrides, adUnitCode, bid, next) {
```

Comment:
Putting the hook here has the disadvantage of not actually suppressing the network call for the bidder. I think it's ok given that it's probably more complex to turn off the bidder completely. We probably just want to point that out it's not a great option to "disable a bidder".